pkg/logging/sumologic/update.go

```diff
@@ -16,10 +16,12 @@ type UpdateCommand struct {
 	common.Base
 	manifest manifest.Data
 
-	Input fastly.GetSumologicInput
-
-	NewName common.OptionalString
+	// required
+	EndpointName string // Can't shaddow common.Base method Name().
+	Version      int
 
+	// optional
+	NewName           common.OptionalString
 	Address           common.OptionalString
```

Comment:
@phamann TODO: this field is extraneous, but not removing as part of this change.

https://developer.fastly.com/reference/api/logging/sumologic/

Reply:
https://github.com/fastly/cli/pull/103/commits/2a0acd2de3e716ffe19c138bab1041bc0ca8252e

Reply:
:+1:

Reply:
I'll pull into a separate PR just for clarity.

Reply:
https://github.com/fastly/cli/pull/104